packages/data-point/lib/entity-types/base-entity/resolve.js

```diff
@@ -23,7 +23,8 @@ function resolveErrorReducers (manager, error, accumulator, resolveReducer) {
   const reducerResolved = resolveReducer(
     manager,
     errorAccumulator,
-    errorReducer
+    errorReducer,
+    [['error']]
```

Comment:
the nested array forces `stringifyReducerStack` to format the output string in a particular way, and I'm using this approach for most of the entity lifecycle reducers

Reply:
Assuming this is kept it might be worth adding a comment about why it's implemented this way.